test/integration/repos_test.rb

```diff
@@ -12,4 +15,34 @@ class ReposTest < ActionDispatch::IntegrationTest
       get '/tgxworld/rails'
     end
   end
+
+  test "query JSON endpoint should return JSON object" do
+    benchmark_type = create(:benchmark_type)
+    benchmark_result_type = create(:benchmark_result_type)
+
+    repo = benchmark_type.repo
+    org = repo.organization
+
+    commit = create(:commit, repo: repo)
+    later_commit = create(:commit, repo: repo, created_at: Time.zone.now + 1.day)
+
+    benchmark_run = create(:commit_benchmark_run,
```

Comment:
they use factory girl which is a fixture replacement, so i'm not sure how using fixture here would interact with that (unless we replace everything with fixtures)